testing/telegraf/APKBUILD

```diff
@@ -0,0 +1,60 @@
+# Contributor: Katie Holly <holly@fuslvz.ws>
+# Maintainer: Katie Holly <holly@fuslvz.ws>
+pkgname=telegraf
+pkgver=1.7.3
+pkgrel=0
+pkgdesc="A plugin-driven server agent for collecting & reporting metrics, part of the InfluxDB project"
+url="https://www.influxdata.com/time-series-platform/telegraf/"
+arch="x86_64 x86"
+license="MIT"
+makedepends="go glide"
+pkgusers="telegraf"
+pkggroups="telegraf"
+install="$pkgname.pre-install"
+options="!strip !net"
+source="$pkgname-$pkgver.tar.gz::https://github.com/influxdata/$pkgname/archive/$pkgver.tar.gz
+	telegraf-makefile-ldflags.patch
+	telegraf.initd
+	telegraf.logrotate
+	glide.yaml
+	glide.lock
+	"
+builddir="$srcdir"/$pkgname-$pkgver
+_godir="$srcdir"/go
+_vendor=influxdata
+
+unpack() {
+	default_unpack
+	mkdir -p "$_godir"/bin "$_godir"/src/github.com/"$_vendor" "$builddir/.glide" "$builddir/.glidetmp"
+	ln -sf "$builddir" "$_godir"/src/github.com/$_vendor/$pkgname
+	cp "$srcdir"/glide.yaml "$srcdir"/glide.lock "$_godir"/src/github.com/$_vendor/$pkgname
+}
+
+build() {
+	cd "$_godir"/src/github.com/$_vendor/$pkgname
+	export GOPATH="$_godir"
+	export PATH="$PATH:$_godir"
+	GLIDE_HOME="$builddir/.glide" GLIDE_TMP="$builddir/.glidetmp" glide install
```

Comment:
good that you set `GLIDE_HOME` so it does not pollute users $HOME, and that you set `GLIDE_TMP` :+1: